himbaechel/himbaechel_dbgen/chip.py

```diff
@@ -229,9 +232,13 @@ def add_bel_pin(self, bel: BelData, pin: str, wire: str, dir: PinType):
 
     def create_wire(self, name: str, type: str="", const_value: str=""):
         # Create a new tile wire of a given name and type (optional) in the tile type
+        gfx_wire_id = 0
+        if (name) in self.gfx_wire_ids:
```

Comment:
what are the brackets here for?

Reply:
ah, that is leftover from past. removing it now